mkt/developers/views.py

```diff
@@ -373,12 +374,32 @@ def content_ratings_edit(request, addon_id, addon):
     }
     request.session.modified = True
 
-    return render(request, 'developers/apps/ratings/ratings_edit.html',
-                  {'addon': addon,
-                   'app_name': get_iarc_app_title(addon),
-                   'form': form,
-                   'company': addon.latest_version.developer_name,
-                   'now': datetime.now().strftime('%Y-%m-%d %H:%M:%S')})
+    ctx = {
+        'addon': addon,
+        'app_name': get_iarc_app_title(addon),
+        'form': form,
+        'company': addon.latest_version.developer_name,
+        'now': datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    }
+
+    if waffle.switch_is_active('iarc-upgrade'):
+        iarc_request_id = IARCRequestID.objects.filter(app_id=addon)
+        if iarc_request_id.exists():
+            iarc_request_id = iarc_request_id[0]
+            outdated = (datetime.now() - iarc_request_id.created >
+                        timedelta(hours=1))
+            if outdated:
+                # IARC Request ID outdated. Re-create.
```

Comment:
I mean the model itself. Not your code: the whole `StoreRequestID` in the spec feels weird to me, I don't understand why we have to create this, things are happening over `https`, with a password, so why do we need to generate and pass a short-lived id ?